### ISSUE TITLE
V2 Ops

### ISSUE THREAD

**Issue by zautumnz** [marxism/marxist.space#6]
This will need a pipeline, maybe Docker build, server set up somewhere, DNS, etc.

**Comment by Nutomic** [comment]
I can create a Docker build no problem, except I dont see any build instructions in the readme or in the repo.

**Comment by zautumnz** [comment]
I have some in the v2 branch. Right now I'm just deploying the repo as is behind Nginx, but having Docker examples might be a nice addition.

**Comment by zautumnz** [comment]
Closing this since it's just about done (just waiting on A records to propagate enough that I can run certbot). I added a separate issue for adding Docker examples.
